src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -781,6 +781,23 @@ protected theorem extractLsb_ofNat (x n : Nat) (hi lo : Nat) :
     (extractLsb' start len x).getLsbD i = (i < len && x.getLsbD (start+i)) := by
   simp [getLsbD, Nat.lt_succ]
 
+@[simp] theorem getMsbD_extractLsb' {start len : Nat} {x : BitVec w} {i : Nat} :
+    (extractLsb' start len x).getMsbD i =
+    (decide (i < len) &&
+    (decide (start + (len - 1 - i) < w) && x.getMsbD (w - 1 - (start + (len - 1 - i))))) := by
```

Comment:
```suggestion
    (extractLsb' start len x).getMsbD i 
    = (decide (i < len) 
       && (decide (start + (len - 1 - i) < w) 
       && x.getMsbD (w - 1 - (start + (len - 1 - i))))) := by
```

I think we currently use both `=` at the end of a line and `=` at the start of the next line styles, but I *personally* find the latter much easier to read; feel free to disagree.